internal/controller/config/config.go

```diff
@@ -95,6 +95,8 @@ func (cm *NumaflowControllerDefinitionsManager) UpdateNumaflowControllerDefiniti
 	// Add or update the controller definition config based on a version
 	for _, controller := range config.ControllerDefinitions {
 		cm.rolloutConfig[controller.Version] = controller.FullSpec
+
+		fmt.Printf("Added/Updated Controller definition Config, version: %s\n", controller.Version)
```

Comment:
shouldn't we use the logger from util package, instead of fmt ?

Reply:
See the PR description. There’s a circular dependency if you try to do that since “logger” module imports “config” module. I tried a little to move directories around to avoid that but it seemed to be complicated. If someone wants to revisit it later, they can.